rotkehlchen/tests/unit/decoders/test_lido_eth.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+
+from rotkehlchen.accounting.structures.balance import Balance
+from rotkehlchen.chain.ethereum.modules.lido.constants import CPT_LIDO_ETH
+from rotkehlchen.chain.evm.constants import ZERO_ADDRESS
+from rotkehlchen.chain.evm.decoding.constants import CPT_GAS
+from rotkehlchen.constants.assets import A_ETH, A_STETH
+from rotkehlchen.fval import FVal
+from rotkehlchen.history.events.structures.evm_event import EvmEvent
+from rotkehlchen.history.events.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.tests.utils.ethereum import get_decoded_events_of_transaction
+from rotkehlchen.types import Location, TimestampMS, deserialize_evm_tx_hash
+
+
+# @pytest.mark.vcr()
```

Comment:
Don't leave commented out code in a PR. This you have to add to our cassettes. I think we discussed how you can do in dev-support right? If not let's chat there.